src/globals/mixins.scss

```diff
@@ -226,6 +226,12 @@
   &.colour-scheme-primary {background-color: $colour-primary; svg {fill: $colour-white;}}
   &.colour-scheme-secondary {background-color: $colour-secondary; svg {fill: $colour-black;}}
   &.colour-scheme-tertiary {background-color: $colour-tertiary; svg {fill: $colour-black;}}
+  &.colour-scheme-brand-1 {background-color: $colour-brand-1; svg {fill: $colour-black;}}
+  &.colour-scheme-brand-2 {background-color: $colour-brand-1; svg {fill: $colour-black;}}
```

Comment:
should this be $colour-brand-2 instead of 1? Similar question for following lines.

Reply:
Good spot. I have updated